src/geometry/GeoParams.cc

```diff
@@ -31,11 +38,41 @@ namespace celeritas
  */
 GeoParams::GeoParams(const char* gdml_filename)
 {
-    CELER_LOG(info) << "Loading from GDML at " << gdml_filename;
     {
         detail::ScopedTimeAndRedirect time_and_output_;
+
+#ifndef CELERITAS_USE_ROOT
```

Comment:
`CELERITA_USE_ROOT` is defined as 0 or 1, rather than being 'undefined' . This protects against misspellings in the macro name and against missing includes, as well as making it easy to use in regular `if` expressions (including the `if constexpr` in C++17).
```suggestion
#if CELERITAS_USE_ROOT
```

Reply:
This is probably another bug which needs to be fixed in vgdml, to make sure the units provided in the GDML file are honored.